packages/action-group/README.md

```diff
@@ -99,6 +99,37 @@ An `<sp-action-group selects="multiple">` will manage its `<sp-action-button>` c
 </sp-action-group>
 ```
 
+## Selected
+
+The `selected` property represents the selection state within a button group. This property can be managed either by the component or by the user. Passing in an array of button values will make `<sp-action-group>` a controllable component.
+
+```html
+<sp-action-group selects="single" selected='["first"]'>
+    <sp-action-button value="first">First</sp-action-button>
+    <sp-action-button value="second">Second</sp-action-button>
+</sp-action-group>
+```
+
+By default, an `<sp-action-group>` will select any button passed into `selected`. Afterwards, `.selects` controls how button values are added to the selection state. For example, if `.selects` is not specified when `selected` is set, any further interaction will result in no change to the selection.
+
+```html
+<sp-action-group selected='["first", "second"]'>
+    <sp-action-button value="first">First</sp-action-button>
+    <sp-action-button value="second">Second</sp-action-button>
+    <sp-action-button value="third">Third</sp-action-button>
+</sp-action-group>
+```
+
+Similarly, if `selected` contains more than one button value, but `selects = "single"`, then those initial buttons will be highlighted, but further interaction will result in radio-button functionality.
+
+```html
+<sp-action-group selects="single" selected='["first", "second"]'>
+    <sp-action-button value="first">First</sp-action-button>
+    <sp-action-button value="second">Second</sp-action-button>
+    <sp-action-button value="third">Third</sp-action-button>
+</sp-action-group>
+```
+
```

Comment:
@hunterloftis @najikahalsema can yall take a look at this copy? I feel like I get it, because I understand what we're going for, but I feel like it's a little different than what we want to be saying about this element.

Separately, should we document a "controlled' usage of this element here?

Reply:
Are we going with a JSON-encoded `"selected"` attribute instead of (or in addition to?) a `selected` property?